migrations/20200213004252-change-date-column-names.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(t => {
+      return Promise.all([
+        queryInterface.removeColumn("events", "description"),
+        queryInterface.removeColumn("events", "date"),
+        queryInterface.addColumn("events", "eventDate", {
+          type: Sequelize.DataTypes.DATE,
```

Comment:
would prefer `DATEONLY` over `DATE` for the column type

Reply:
when I'm doing this change, should I create a new migration? Or can I update the existing one?

Reply:
since we're already asking people to re-run migrations for this case, you can make the update to this existing migration - this will not be the correct pattern moving forward though